### PR TITLE
Please check

### PR DESCRIPTION
**Pull request by Oonishi-Jungo**

### CODE REVIEW

File: OJ_test.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+abcdefg
+hijklmn
```
Comment (Seinobu) [inline OJ_test.txt:2]: LOL!

Comment (hikarsat) [reply OJ_test.txt:2]: RGB!

Comment (MtsuYas) [reply OJ_test.txt:2]: good game

Comment (k-hiromoto) [reply OJ_test.txt:2]: OK!

**Review by k-hiromoto — APPROVED** [review]

**Review by MtsuYas — APPROVED** [review]

**Review by hikarsat — APPROVED** [review]
承認します

**Pull request merged**